### PR TITLE
CD release notes 792xx

### PR DESCRIPTION
**Pull request by roshnisarangadharan**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

**Review by thisrohangupta — DISMISSED** [review]
These changes look good, thanks

File: release-notes/continuous-delivery.md [hunk 1]
```diff
@@ -15,12 +15,161 @@ Review the notes below for details about recent changes to Harness Continuous De
...
+- The environment reference value in the service override configuration has been migrated to a scoped reference. (CDS-58214)
+  
+  When you create or update a new service override, the environment reference was an identifier before. Now, Harness supports scoped environment references in metadata as well as YAML as part of the overrides enhancement.
+- The pipeline variables, stage variables, and service and environment variables are no longer validated on the run pipeline form. These validations are optional now. (CDS-64656)
+
+```mdx-code-block
+  </TabItem>
+  <TabItem value="Early access">
+```
+- Added support to get GitHub webhook events through a CURL command if the client call fails. (CDS-58692)
```
Comment (michaelcretzman) [inline release-notes/continuous-delivery.md:109]: how is this an Early Access feature? Is it behind a feature flag?

I don't know what the summary means so I've asked the dev in the ticket

File: release-notes/continuous-delivery.md [hunk 2]
```diff
@@ -15,12 +15,161 @@ Review the notes below for details about recent changes to Harness Continuous De
...
+                type: String
+                value: stage3Var
+
+  ```
+  
+  
+- A warning pops up when you create a new template with already existing identifiers in the same scope using a new version label. You can choose to merge the template with the existing template by selecting the **Save as new version of existing template** button in the pop-up window. (CDS-47301)
+- The environment reference value in the service override configuration has been migrated to a scoped reference. (CDS-58214)
+  
+  When you create or update a new service override, the environment reference was an identifier before. Now, Harness supports scoped environment references in metadata as well as YAML as part of the overrides enhancement.
```
Comment (michaelcretzman) [inline release-notes/continuous-delivery.md:102]: Update: Rohan said to remove this one:
> the story is getting deployed but the epic and enhancement isnt done, no need to document it

**Review by michaelcretzman — CHANGES_REQUESTED** [review]

File: release-notes/continuous-delivery.md (hunk 2 above)
Comment (roshnisarangadharan) [reply release-notes/continuous-delivery.md:105]: ok

File: release-notes/continuous-delivery.md (hunk 1 above)
Comment (roshnisarangadharan) [reply release-notes/continuous-delivery.md:112]: Moved to new features list

File: release-notes/continuous-delivery.md [hunk 3]
```diff
@@ -15,12 +15,162 @@ Review the notes below for details about recent changes to Harness Continuous De
...
+  
+  Added support for SSH URL in triggers.
+- Resolved an issue that converted runtime fields with default values to fixed values when viewing the template-linked parts of a pipeline like steps, stage, and so on. (CDS-67999)
+- Users were able to edit the **Freeze Windows>** **Schedule** tab when a freeze window was active. (CDS-58507)
+  
+  The **Schedule** tab is not editable anymore when a freeze window is active. It is also uneditable for users with read-only permissions. 
+- Fixed minor bugs in the Bamboo artifact source connector. (CDS-58632)
+- The Security Testing Orchestration (STO) module was present in the Harness CD Community Edition (CE). (CDS-59269)
+  
+  Support has been added to hide the STO module from CE.
```
Comment (charanya-j) [inline release-notes/continuous-delivery.md:149]: Added support to

**Review by charanya-j — DISMISSED** [review]

**Pull request merged**